app/models/reports/patient_follow_up.rb

```diff
@@ -4,6 +4,11 @@ class PatientFollowUp < Reports::View
     belongs_to :patient
     belongs_to :facility, class_name: "::Facility"
     belongs_to :user
+    enum patient_gender: {
```

Comment:
Should this use `Rails.application.config.country[:supported_genders]` like it does in https://github.com/simpledotorg/simple-server/blob/master/app/models/patient.rb?

Reply:
I wanted to keep it simple and have the same enum across everywhere here in the reporting pipeline.  All the UI side gender pieces will still validate and enforce the `supported_genders` tho.